### ISSUE TITLE
Перевести console

### ISSUE THREAD

**Issue by Olegas** [iojs/iojs-ru#30]

**Comment by prog666** [comment]
начал и закончил

**Comment by Olegas** [comment]
Извините, а вы его машинным переводом что-ли переводили?

**Comment by prog666** [comment]
никак нет, могу больше не переводить если совсем печально..

**Comment by Olegas** [comment]
Нет, не печально. Просто некоторые предложения странно сформулированы. Показалось что машинный.

**Comment by Olegas** [comment]
@iojs/iojs-ru давайте смотреть
